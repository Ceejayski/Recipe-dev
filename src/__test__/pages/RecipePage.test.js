import { Provider } from 'react-redux';
import { create } from 'react-test-renderer';
import RecipePage from '../../pages/recipePage';
import store from '../../redux';

test('renders Correctly', () => {
  const Placeholder = () => (
    <Provider store={store}>
      <RecipePage />
    </Provider>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
