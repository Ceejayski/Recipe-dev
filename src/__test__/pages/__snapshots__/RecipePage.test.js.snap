// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
<div
  className="container"
  style={
    Object {
      "minHeight": "100vh",
    }
  }
/>
`;
